### PR TITLE
feat: add binary support for verifiers

### PR DESCRIPTION
**Pull request by ffe9f8**
Based on #991 by @talalashraf 
Co-authored by @StephenFluin (tab support)

- Split options and steps for setting up and configuring `tofnd` and `ampd` into different sections for clarity.

Preview: https://axelar-docs-git-marty-add-binary-support-axelar-network.vercel.app/validator/amplifier/verifier-onboarding

### CODE REVIEW

File: src/pages/validator/amplifier/verifier-onboarding.mdx [hunk 1]
```diff
@@ -8,45 +8,106 @@ The Axelar Virtual Machine (AVM) and Amplifier are currently under active develo
...
+<tabs>
+
+<tab-item title="Binary">
+Download the [`tofnd` binary](https://github.com/axelarnetwork/axelar-amplifier/releases/tag/ampd-v0.4.0) depending on the type of machine you have:
+    - Linux: [`tofnd-linux-amd64-v0.10.5`](https://github.com/axelarnetwork/tofnd/releases/download/v0.10.5/tofnd-linux-amd64-v0.10.5)
+    - Apple Silicon Mac: [`tofnd-darwin-arm64-v0.10.5`](https://github.com/axelarnetwork/tofnd/releases/download/v0.10.5/tofnd-darwin-arm64-v0.10.5)
+    - Intel Mac: [`tofnd-darwin-amd64-v0.10.5`](https://github.com/axelarnetwork/tofnd/releases/download/v0.10.5/tofnd-darwin-amd64-v0.10.5)
+</tab-item>
+
+<tab-item title="Docker">
```
Comment (StephenFluin) [inline src/pages/validator/amplifier/verifier-onboarding.mdx:33]: Source first, add missing source instructions

File: src/pages/validator/amplifier/verifier-onboarding.mdx [hunk 2]
```diff
@@ -8,45 +8,106 @@ The Axelar Virtual Machine (AVM) and Amplifier are currently under active develo
...
-        
-    You should make the binary executable with `chmod a+x ./ampd-darwin-arm64-v0.4.0`, customizing the binary name to match the one you downloaded.
+All instructions in this tutorial will assume you have a binary in your `PATH` named `ampd`. If you don't want to add `ampd` to your `PATH`, you can refer to the binary directly in every command in this tutorial, replacing `ampd` with the `ampd` version that corresponds with your machine.
 
-    If you don't want to add it to your path, you can refer to the binary directly in every command in this tutorial, such as `~/ampd-linux-amd64-v04.0 <command>`.
+For example, to determine the verifier address without the `ampd` alias on an Apple Silicon Mac, you would run:
 
-    If you get a security warning while trying to run `ampd` on Mac, you can disable the [gatekeeper](https://support.apple.com/guide/security/gatekeeper-and-runtime-protection-sec5599b66df/web) for a single binary with `sudo xattr -d com.apple.quarantine ./ampd-darwin-arm64-v0.4.0` or disable gatekeeper globally with `sudo spctl --master-disable`.
+```bash
+~/ampd-darwin-arm64-v0.4.0 verifier-address
```
Comment (StephenFluin) [inline src/pages/validator/amplifier/verifier-onboarding.mdx:101]: This needs to be configured per type (source, docker, binary)

Comment (StephenFluin) [reply src/pages/validator/amplifier/verifier-onboarding.mdx:101]: eg when I run commands with docker, I'll need to:
`docker run axelarnet/axelar-ampd:v0.4.0 verifier-address`

**Review by StephenFluin — APPROVED** [review]

**Comment by ffe9f8** [comment]
Updates in #1015

**Pull request merged**